os.kieserver.s2i/module.yaml

```diff
@@ -2,5 +2,8 @@ schema_version: 1
 name: os.kieserver.s2i
 version: '1.0'
 description: Legacy os.kieserver.s2i script package.
+envs:
+- name: MAVEN_LOCAL_REPO
```

Comment:
@rcernich Should this env also be added to our 7.1 kieserver image?
This is the associated module in 7.1 that contains the s2i scripts:
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/module.yaml

Reply:
You'll probably need this once CLOUD-2477 is merged.  The default repo is now /tmp/artifacts/m2, which aligns better with save-artifacts and incremental builds, as we don't have to copy things around in the image.